### PR TITLE
Rename `keep_dim` keyword name to `keepdim`

### PR DESCRIPTION
**Pull request by rdspring1**
* Match pytorch name scheme like `torch.sum(input, dim, keepdim=False, *, dtype=None)`

### CODE REVIEW

**Comment by rdspring1** [comment]
!test
